tests/test_house_creation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from house.models import House, Job
+
+
+@pytest.mark.django_db()
+class TestHouseCreation:
+    def test_good_form(self, client, new_user):
```

Comment:
The name of the test is not entirely clear

Reply:
what is not clear about Testing The House Creation?
and when you comment about the clarity of name please suggest a better one or check google for the english word if you do not know it, or better explain what is not clear.